cluster/config-defaults.yaml

```diff
@@ -100,8 +100,10 @@ skipper_readiness_init_delay_seconds: 1
 skipper_liveness_init_delay_seconds: 30
 {{end}}
 # skipper termination settings
-skipper_termination_grace_period: "70"
-skipper_wait_for_healthcheck_interval: "65s"
+# grace period needs to be >timeout
+# wait for healtcheck needs to be >grace period
+skipper_termination_grace_period: "355"
+skipper_wait_for_healthcheck_interval: "360s"
```

Comment:
I think `skipper_wait_for_healthcheck_interval` should be less then `skipper_termination_grace_period` otherwise k8s will always terminate Skipper before it shutdowns itself

Reply:
I will test it again, but it was tested in this config ~1 year ago with success.

Reply:
Reviewing my own document showed that you are right :D

Reply:
https://github.com/zalando-incubator/kubernetes-on-aws/pull/5024